docker/Dockerfile.release

```diff
@@ -35,6 +35,10 @@ RUN mvn org.apache.maven.plugins:maven-dependency-plugin:get \
 RUN mkdir -p /zipkin && cp ~/.m2/repository/io/zipkin/zipkin-server/${version}/zipkin-server-${version}-exec.jar /zipkin && cd /zipkin && jar xf *.jar && rm *.jar
 RUN mkdir -p /zipkin-slim && cp ~/.m2/repository/io/zipkin/zipkin-server/${version}/zipkin-server-${version}-slim.jar /zipkin-slim && cd /zipkin-slim && jar xf *.jar && rm *.jar
 
+# statically evaluate classpath to avoid https://github.com/docker/for-mac/issues/3643
+RUN cd /zipkin && echo .:$(ls ${PWD}/BOOT-INF/lib/*.jar|tr '\n' ':')${PWD}/BOOT-INF/classes > classpath
+RUN cd /zipkin-slim && echo .:$(ls ${PWD}/BOOT-INF/lib/*.jar|tr '\n' ':')${PWD}/BOOT-INF/classes > classpath
```

Comment:
hmm I think this won't be correct, because PWD will be /zipkin-slim where we want it to end up /zipkin later..

Reply:
Oops - thanks for noticing I only tested fat version ;) Fixed to prepend `/zipkin/` to relative paths

Reply:
hey hey hey... not slim != fat.. more like slim++